### PR TITLE
Adapt to desktop

### PR DESCRIPTION
**Pull request by ChrispaixK**
# Microverse_Portfolio: desktop version

In this PR I :
* Add media queries for desktop version in CSS file
* Adapt the mobile version to desktop screens
* Made images flexible
* Add navigation menu links in HTML
* Add id selectors on sections in the HTML file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -213,6 +218,7 @@ <h3>Let's connect</h3>
         <ul class="ul-1">
             <li class="li-1">
                 <h3 class="li-1_title">languages <i class='fa fa-chevron-down'></i></h3>
+                <p></p>
```
Comment (roshan-bajgain) [inline index.html:221]: - [x] You must replicate the design as much as possible (e.g., font, colors, images, tests, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- [x] Your drop-down menu should be at the right of the page as the Figma design. Kindly implement it.
### Your design
his
### Figma design
figma

File: style.css [hunk 2]
```diff
@@ -413,3 +420,165 @@ textarea {
...
+    list-style-type: none;
+    display: flex;
+    margin-left: 25px;
+  }
+
+  .links li > a {
+    color: #172b4d;
+    text-decoration: none;
+    text-transform: capitalize;
+  }
```
Comment (roshan-bajgain) [inline style.css]: - [x] Kindly use this `color: #344563` to make the design similar to Figma.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @ChrispaixK, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️

## Required Changes ♻️

- [x] [Optional] You have only one commit kindly write more commits and try to write a good commit message that makes your project better.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ChrispaixK** [comment]
Thank you @roshan-bajgain

**Review by rustam-tolipov — APPROVED** [review]
## Hi @ChrispaixK 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**